TaskHero/Step.swift

```diff
@@ -8,21 +8,23 @@
 
 import Foundation
 
-enum StepState: Int {
-    case NotStarted = 0
-    case InProgress = 1
-    case Completed = 2
+class StepState: NSObject {
```

Comment:
@akishinagawa just for your fyi, I changed this to a string because Parse was rejecting the enum (I think it's because the state below was of type StepState), and didn't bother to debug too much.